input/ig.xml

```diff
@@ -221,23 +221,23 @@
 				<reference value="AllergyIntolerance/lactose"/>
 			</reference>
 			<name value="AllergyIntolerance - Lactose intolerance"/>
-			<description value="Lactose (food) intolerance&#xa;&#xa;Patient: Mia BANKS"/>
+			<description value="Shows an example for the *AU Core AllergyIntolerance* profile.&#xa;&#xa;Patient: Mia BANKS"/>
```

Comment:
Don't linebreak Patient. Apply to all example markdown.

Reply:
Done